.github/workflows/daily-check.yml

```
@@ -10,4 +10,4 @@ jobs:
       - uses: actions/checkout@v3
       - run: bash check-new-release
         env:
-          GITHUBTOKEN: ${{ github.token }}
+          GITHUBTOKEN: ${{ secrets.github_token }}
```

May I suggest a name that recalls WHO owns the token?

```suggestion
          GITHUBTOKEN: ${{ secrets.nethbot_token }}
```